src/main/java/org/jabref/gui/collab/EntryDeleteChangeViewModel.java

```diff
@@ -37,6 +39,7 @@ public void makeChange(BibDatabaseContext database, NamedCompound undoEdit) {
     @Override
     public Node description() {
         PreviewViewer previewViewer = new PreviewViewer(new BibDatabaseContext(), JabRefGUI.getMainFrame().getDialogService(), Globals.stateManager, Globals.getThemeManager());
```

Comment:
Instead of initializing this in the subclasses I would move it to the parent class and completely remove this method here and in the other subclasses.